replicate/configmap/configmaps.go

```diff
@@ -237,7 +239,8 @@ func (r *Replicator) PatchDeleteDependent(sourceKey string, target interface{})
 	logger.Debugf("clearing dependent config map %s", dependentKey)
 	logger.Tracef("patch body: %s", string(patchBody))
 
-	s, err := r.Client.CoreV1().ConfigMaps(targetObject.Namespace).Patch(targetObject.Name, types.JSONPatchType, patchBody)
+	// TODO need to check on this
+	s, err := r.Client.CoreV1().ConfigMaps(targetObject.Namespace).Patch(context.TODO(), targetObject.Name, types.JSONPatchType, patchBody, metav1.PatchOptions{}, "")
```

Comment:
IIRC, the `subresources` parameter is a variadic `...string` parameter that you should be able to omit entirely if you do not want to patch a particular subresource...? 🤔 

```suggestion
	s, err := r.Client.CoreV1().ConfigMaps(targetObject.Namespace).Patch(context.TODO(), targetObject.Name, types.JSONPatchType, patchBody, metav1.PatchOptions{})
```

Reply:
thanks!